source/_components/folder_watcher.markdown

```diff
@@ -13,16 +13,23 @@ ha_iot_class: "Local Polling"
 ha_release: 0.67
 ---
 
-This component adds [Watchdog](https://pythonhosted.org/watchdog/) file system monitoring, publishing events on the Home-Assistant bus on the creation/deletion/modification of files.
+This component adds [Watchdog](https://pythonhosted.org/watchdog/) file system monitoring, publishing events on the Home Assistant bus on the creation/deletion/modification of files. The monitored `event_type` are:
+* `created`
+* `deleted`
+* `modified`
+* `moved`
 
-To configure the `folder_watcher` component add to you `configuration.yaml` file:
+Note that by default folder monitoring is recursive, meaning that the contents of sub-folders are also monitored.
+
+To enable the Folder Watcher component in your installation, add the following to your `configuration.yaml` file:
 
-```yaml
 {% raw %}
+```yaml
 folder_watcher:
-  - folder: /config
-{% endraw %}
+  watchers:
```

Comment:
Based on the code, this is wrong? https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/folder_watcher.py#L22-L28

Reply:
Good catch, don't know how that snuck back in